src/views/main-view/main-sidebar/MainSidebar.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div class="left-nav">
+    <communities-list></communities-list>
+    <bottom-section></bottom-section>
+  </div>
+</template>
+
+<script lang="ts">
+import CommunitiesList from "./CommunitiesList.vue";
+import BottomSection from "./BottomSection.vue";
+
+import { useStore } from "vuex";
+import { defineComponent, ref } from "vue";
+import { CommunityState } from "@/store";
+
+export default defineComponent({
+  components: { CommunitiesList, BottomSection },
+  setup() {
+    const store = useStore();
+
+    const joiningLink = ref("");
+
+    const newCommunityName = ref("");
+    const newCommunityDesc = ref("");
+
+    const showModal = ref(false);
+    const isCreatingCommunity = ref(false);
+    const tabView = ref("Create");
+
+    const changeCommunity = (community: CommunityState) => {
+      store.commit("changeCommunity", community);
+    };
+
+    const handleCommunityClick = (community: CommunityState) => {
```

Comment:
`changeCommunity` function does that same thing

Reply:
Fixed. This was not in use anymore, as we are using vue-router for navigation.